app/models/lettings_log.rb

```diff
@@ -45,31 +45,58 @@ class LettingsLog < Log
   scope :filter_by_location_postcode, ->(postcode_full) { left_joins(:location).where("REPLACE(locations.postcode, ' ', '') ILIKE ?", "%#{postcode_full.delete(' ')}%") }
   scope :search_by, lambda { |param|
     filter_by_location_postcode(param)
-        .or(filter_by_tenant_code(param))
-        .or(filter_by_propcode(param))
-        .or(filter_by_postcode(param))
-        .or(filter_by_id(param))
+      .or(filter_by_tenant_code(param))
+      .or(filter_by_propcode(param))
+      .or(filter_by_postcode(param))
+      .or(filter_by_id(param))
   }
   scope :filter_by_before_startdate, ->(date) { where("lettings_logs.startdate >= ?", date) }
   scope :unresolved, -> { where(unresolved: true) }
-
   scope :filter_by_organisation, ->(org, _user = nil) { where(owning_organisation: org).or(where(managing_organisation: org)) }
   scope :filter_by_owning_organisation, ->(owning_organisation, _user = nil) { where(owning_organisation:) }
   scope :filter_by_managing_organisation, ->(managing_organisation, _user = nil) { where(managing_organisation:) }
-
+  scope :age1_answered, -> { where.not(age1: nil).or(where(age1_known: 1)) }
```

Comment:
Could this be done via a helper? or by looking to another class? if feels wrong having it hard coded on the model, though I can't articulate why. Is this code copied and pasted anywhere else that could also depend on a helper instead?.

Reply:
I agree it does seem a bit hard codey, but I'm struggling to think of a way we could make it more dynamic (like a general `question_answered(id:)` method) and still be useful in other cases given there's variation between different fields and their `_known` fields' values for yes and no - if it's not something we need to generalise I think this is ok.

Reply:
I think for me this is mostly there so that the query runs faster, and it's separated out so that it's a bit more readable